netkan/netkan/cli.py

```diff
@@ -226,12 +226,16 @@ def redeploy_service(cluster, service_name):
 @click.option(
     '--days', help='Purge items older than X from cache',
 )
-def clean_cache(days):
+@click.option(
+    '--cache', envvar='NETKAN_CACHE', default=str(Path.home()) + '/ckan_cache/',
+    help='Absolute path to the mod download cache',
+)
+def clean_cache(days, cache):
     older_than = (
```

Comment:
Neat! I like this, same comment as above. If you want to play around with setting a type here, that could be interesting (but I will happily merge without it).

https://click.palletsprojects.com/en/7.x/api/?highlight=click%20path#click.Path

example use here:

https://click.palletsprojects.com/en/7.x/options/?highlight=click%20path#multiple-values-from-environment-values

Reply:
Cool! Probably only useful for unit tests right now, as we don't run an integration test with the values we use in production, so we can't catch errors in the `prod-stack.yml`. But better than nothing.